src/components/editor/metaform-editor-right-drawer-feature.tsx

```diff
@@ -853,6 +853,9 @@ const MetaformEditorRightDrawerFeature: FC<Props> = ({
         })
         }
       />
+      <Typography variant="body2">
+        { `${strings.draftEditorScreen.editor.features.field.fieldType}: ${field.type}` }
```

Comment:
field.type should be localized. Use same values that are used in left side drawer fields tab.